libtenzir/builtins/contexts/lookup_table.cpp

```diff
@@ -379,9 +419,11 @@ class ctx final : public virtual context {
 
   auto reset() -> caf::expected<void> override {
     context_entries.clear();
+    subnet_entries.clear();
     return {};
   }
 
+  // TODO: persist patricia trie.
```

Comment:
@tenzir/pipelines-squad does this requires bumping the Flatbuffer version?

Reply:
Why would you need to persist it at all? All the data is already present in the lookup table, so when loading you can just recreate the trie.